src/composer.json

```diff
@@ -16,10 +16,13 @@
         "stof/doctrine-extensions-bundle": "^1.3",
         "symfony/asset": "4.3.*",
         "symfony/console": "4.3.*",
+        "symfony/debug-pack": "^1.0",
         "symfony/flex": "^1.3.1",
         "symfony/form": "4.3.*",
         "symfony/framework-bundle": "4.3.*",
+        "symfony/maker-bundle": "^1.12",
         "symfony/orm-pack": "^1.0",
+        "symfony/routing": "4.3.*",
```

Comment:
Is still still needed with the routing yaml files ?

Reply:
Yep, it helps in debugging